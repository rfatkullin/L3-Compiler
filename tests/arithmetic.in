func main (args->[[char]]) -> int
	
	print(1 + 1 + 1);
	print(1 2 1);
	print(1 2 1 0);
	
	print(12 / 2 / 3);

	print(10 % 100 % 1000);

	print(2^3^3);

	print((1 + 1 + 1) (2 + 2^10));

	print(10 - 5 - 2);

	print((2 + 6) / 4 - 1);

	print(99 + 6 / 2);

	print(2 + 2 - 2);

	print(2 2^10 / 3 % 10 + 2 - 1);

	print(4 5 / 10 50);
	
	main = 0;
endfunc