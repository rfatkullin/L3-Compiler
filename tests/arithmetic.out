3
2
0
2
10
134217728
3078
3
1
102
2
3
